lib/display.rb

```diff
@@ -1,108 +1,28 @@
 class Display
-  def start_menu
-    puts <<~START_MENU
-      ---------------------------------------------------
-      |          WELCOME TO TOY ROBOT SIMULATOR         |
-      ---------------------------------------------------
-
-
-      First by starting up your board!
-    START_MENU
-  end
-
-  def parse_input_to_integer
-    input = gets.chomp
-    if input == 'exit'
-      exit(true)
-    elsif input.scan(/\D/).any?
-      input
-    else
-      input.to_i
-    end
-  end
-
-  def get_board_rows
-    puts 'Enter the amount of rows you want to have for your board:'
-    parse_input_to_integer
+  def exit_or_proceed(input)
+    input == 'exit' ? exit(true) : input
   end
 
-  def get_board_columns
-    puts 'Enter the amount of columns you want to have for your board:'
-    parse_input_to_integer
+  def draw(screen)
+    puts '---------------------------------'
+    puts screen
+    puts '---------------------------------'
   end
 
-  def get_robot_start_row
-    puts 'Which row do you want to place your robot:'
-    parse_input_to_integer
+  def notify(message)
+    puts message
```

Comment:
This feels a bit weird looking at it 😅 because I could just run `puts` on the `App` class itself, but then if I do that, it does not show much of ` Display` responsibilities if it have getting user input functionality.